SConstruct

```diff
@@ -251,11 +285,15 @@ elif env["platform"] == "ios":
     env["AR"] = compiler_path + "ar"
     env["RANLIB"] = compiler_path + "ranlib"
 
-    env.Append(CCFLAGS=["-std=c++14", "-arch", env["ios_arch"], "-isysroot", sdk_path])
+    if env["arch"] == "arm32":
+        env.Append(CCFLAGS=["-arch", "armv7"])
+        env.Append(LINKFLAGS=["-arch", "armv7"])
+    else:
+        env.Append(CCFLAGS=["-arch", env["arch"]])
+        env.Append(LINKFLAGS=["-arch", env["arch"]])
+    env.Append(CCFLAGS=["-std=c++14", "-isysroot", sdk_path])
     env.Append(
```

Comment:
Adding `universal` option for iOS (`x86_64` + `arm64` for simulator, and `armv7` + `arm64` for device) would be also useful.

Reply:
Good idea, done.